monai/networks/nets/resnet.py

```diff
@@ -14,6 +14,10 @@
 from collections.abc import Callable
 from functools import partial
 from typing import Any
+from pathlib import Path
+import logging
+import re
+from huggingface_hub import hf_hub_download
```

Comment:
May use `optional_import` here and also in the unit test.
https://github.com/Project-MONAI/MONAI/blob/dev/CONTRIBUTING.md#adding-new-optional-dependencies

Reply:
Thanks for your feedback. I'll fix it.